pkg/secrethub/http.go

```diff
@@ -21,10 +21,12 @@ var (
 const (
 	baseURLPath = "/v1"
 
+	pathAuthenticate = "%s/auth"
+
 	// Current account
 	pathMeUser              = "%s/me/user"
 	pathMeRepos             = "%s/me/repos"
-	pathMeKey               = "%s/me/key"
+	pathMeKey               = "%s/me/key?key_version=v2"
```

Comment:
I'd stay away from manipulating the path here. Not 100% sure, but it does assume this path is an absolute path and cannot be joined with anything else. Although, the other paths are that as well. Think about it and maybe just ignore my ramblings here.

Reply:
I think we notice it soon enough if this leads to problems.